<launch>

	<!-- <include file="$(find aruco_pose)/launch/CupAndAruco.launch">
	</include> -->

	<include file="$(find mission)/launch/little.launch" >
	</include>

	<node name="eurobot_gui" pkg="eurobot_gui" type="eurobot_gui.py" output="screen" respawn="false">
	</node>

	<node name="data_node" pkg="main2021" type="Data_small" output="screen" respawn="false">
	</node>

	<node name="small_goap_node" pkg="main2021" type="goap_little_server.py" output="screen" respawn="false">
	</node>

	<node name="camera_node" pkg="main2021" type="cameraTest" respawn="false">
	</node>

	<!--<node name="small_main_node" pkg="main2021" type="small_main" output="screen" respawn="false">
	</node> -->

	<node name="small_main_node" pkg="main2021" type="small_main_demo0326" output="screen" respawn="false">
	</node>

	<param  name="start" type="int" value="0"/>
</launch>

